infrastructure/main.tf

```diff
@@ -13,11 +13,11 @@ locals {
   db_name = "opal"
```

Comment:
This needs to be updated too - this is what actually names the DB in higher environments

Reply:
done